### ISSUE TITLE
Long time to run with TALC

### ISSUE THREAD

**Issue by CrazyHsu** [lbroseus/TALC#1]
Hi,
I have followed your instruction to run my data with TALC, but the running time of the program have reached 511,898 hours with 30 threads and it seems like that it don't want to stop. As mentioned in your document, I trimmed my data using [fastp](https://github.com/OpenGene/fastp), which can detected adapters and trim them automatically, with the params ```-q 20 -l 150```. Can you give some suggestions how to reduce the time it costs? Thank you.

**Comment by lbroseus** [comment]
Hi,
First of all, could you give me more information about your datasets: 
which organism has been sequenced? which sequencing technologies were used? how many reads are there in your long and short read datasets?

**Comment by CrazyHsu** [comment]
Thank you for prompt reply. 
My organism is maize, and PacBio iso-seq and Illumina paired-end 150bp sequencing has been applied which generated 1,592,504 full-length reads and 14.5G short reads(left 7.2G, right 7.3G). Then I trimmed the short reads with fastp, and applied it to 1,592,504 full-length reads with TALC. It turns out to be time-consuming. Should I cluster my full-length reads into collapsed transcripts?

**Comment by lbroseus** [comment]
Hi,
Your datasets are not that large, yet. 
Do you perform the computations on a busy server?
Is it better if your reduce the number of threads, to 15 for example?

**Comment by CrazyHsu** [comment]
Nope, there is no other computation intensive program in my server, that can't be a reason for slow execution. And why can it be better if I reduce the number of threads? Shouldn't it be the more threads used, the fast it runs?
